### ISSUE TITLE
Remove hardcoded parameters

### ISSUE THREAD

**Issue by manuelli** [RobotLocomotion/pytorch-dense-correspondence#104]
- [ ] `M_descriptor` in the loss function [here](https://github.com/peteflorence/pytorch-dense-correspondence/pull/102/files#diff-aa26a5370806c29d4f3930207263110fR340) and [here](https://github.com/peteflorence/pytorch-dense-correspondence/pull/102/files#diff-aa26a5370806c29d4f3930207263110fR349)
